azafea/event_processors/endless/metrics/events/__init__.py

```diff
@@ -458,7 +458,9 @@ class LocationLabel(SingularEvent):
 
     @staticmethod
     def _get_fields_from_payload(payload: GLib.Variant) -> Dict[str, Any]:
-        return {'info': get_asv_dict(payload)}
+        # Empty values are removed as they are useless, even if they are sent
+        # by old versions of eos-metrics-instrumentation
+        return {'info': {key: value for (key, value) in get_asv_dict(payload).items() if value}}
```

Comment:
I don't think we want to store the row at all if it contains no useful information.

Reply:
OK!

Reply:
I can’t find an easy way to fully ignore the line if it’s empty.

The best solution I’ve found is to raise an `EmptyPayloadError` that will be catched by [`new_singular_event`](https://github.com/endlessm/azafea/blob/7fea174cefe4b29df4719bef149c49acc5a147c7/azafea/event_processors/endless/metrics/events/_base.py#L262). If the event is added to `IGNORED_EMPTY_PAYLOAD_ERRORS`, the line will silently be ignored (as far as I can understand).

Is there another better way?

Reply:
That indeed seems to be the only currently-implemented way, and I don't see any obvious refactorings that would be clearer.

Reply:
Done.